Alarmify/Modules/Edit/SubViews/FeaturesListVIew.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  FeaturesListVIew.swift
+//  Alarmify
+//
+//  Created by Swastik Patil on 04/06/24.
+//
+
+import SwiftUI
+import AudioToolbox
+
+struct FeaturesListVIew: View {
+    @EnvironmentObject var alarmdata: AlarmData
+    var body: some View {
+        NavigationStack {
```

Comment:
You shouldn't have more than one navigation stack or navigation view at one screen. Usually NavigationStack is the root container view of the screen.

Reply:
I thought in the figma file there was only dark mode so I though it would be dark mode only, Sorry my bad, Changed it